website/src/content/docs/start_here/install.md

```diff
@@ -0,0 +1,52 @@
+---
+title: SCAAML Installation
+description: SCAAML software installation
+---
+
+Installation of the SCAAML Python package.
+
+## Dependencies
+
+### [Optional] PicoScope &reg; 6424E
+
+To use the Python oscilloscope control of a [PicoScope
+&reg;](https://www.picotech.com/products/oscilloscope) 6424E one needs to
+install the corresponding libraries.  If you do not plan to use this the only
+downside of no having these libraries is some failing unit-tests.  Follow the
+official guide
+[https://www.picotech.com/downloads/linux](https://www.picotech.com/downloads/linux)
+or on a Debian based system one can just:
+
+```bash
+sh -c 'wget -qO - https://labs.picotech.com/Release.gpg.key | sudo apt-key add -'
+sudo sh -c 'echo "deb https://labs.picotech.com/picoscope7/debian/ picoscope main" >/etc/apt/sources.list.d/picoscope7.list'
+apt-get update && apt-get install -y udev usbutils
+mv /sbin/udevadm /sbin/udevadm.bin ; echo '#!/bin/bash' > /sbin/udevadm && chmod a+x /sbin/udevadm
```

Comment:
1. touching a sbin binary likely requires sudo
2. why?

Reply:
Docker config (both sbin and absense of sudo). Leaving just "follow official documentation"